web/src/util/serialization/apd.js

```diff
@@ -3,6 +3,19 @@ import {
   fromAPI as activityFromAPI
 } from './activities';
 import { replaceNulls } from '../index';
+import assurancesList from '../../data/assurancesAndCompliance.yaml';
+
+export const initialAssurances = Object.entries(assurancesList).reduce(
+  (acc, [name, regulations]) => ({
+    ...acc,
+    [name]: Object.keys(regulations).map(reg => ({
+      title: reg,
+      checked: false,
+      explanation: ''
+    }))
+  }),
+  {}
+);
```

Comment:
We should think about this one a little bit, but right now, by design, the assurances list is entirely client-driven.  It's a JSON field on the backend, so it defaults to empty but will happy accept *whatever*.  Since the frontend depends on these values, it made sense to me for the frontend to create them.

This feels acceptable to me right now, but interested in what you think.

Reply:
this makes sense to me